# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/ncompress/config.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./build.orig	1992-10-28 12:10:53.000000000 +0100
+++ ./build	2004-08-06 22:38:17.586220888 +0200
@@ -1,4 +1,3 @@
-:
 #!/bin/sh
 #
 version="4.2.4"
@@ -19,6 +18,7 @@
 trap 'set +x ; exit 1' 1 2 15
 
 [ -f compress.def ] && . ./compress.def
+x=1
 
 while true
 do
@@ -70,7 +70,9 @@
 			done
 		) >compress.def
 
-		read choice var1 var2 var3 var4 varr
+		read choice var1 var2 var3 var4 varr < /dev/null
+		if [ $x = 1 ]; then choice=genmake; x=2
+		else choice=q; fi
 		echo ""
 	else
 		choice=D
@@ -278,8 +280,6 @@
 		[ -f /usr/include/sys/dir.h ] && { SYSDIR=yes; }
 		[ -f /usr/include/dirent.h ] && { DIRENT=yes; }
 		[ -f /usr/include/utime.h ] && { UTIME_H=yes; }
-		[ -d /usr/local/bin ] && { BINDIR=/usr/local/bin; }
-		[ -d /usr/local/man ] && { BINDIR=/usr/local/man/man1; }
 		[ -f /usr/bin/compress ] && { BINDIR=/usr/bin; }
 	
 		if [ ".${CC}" = . ]
@@ -790,7 +790,7 @@
 
 
 compress:	Makefile compress42.c patchlevel.h
-	$(CC) -o compress $(options) "-DCOMPILE_DATE=\"\`date\`\"" compress42.c $(LBOPT)
+	\$(CC) -o compress \$(options) "-DCOMPILE_DATE=\\"\`date\`\\"" compress42.c \$(LBOPT)
 
 install:	compress
 		[ -f \$(BINDIR)/compress ] && \\
